postgres/Dockerfile

```diff
@@ -1,7 +1,51 @@
-FROM postgres:15.1
+FROM ubuntu:22.04
```

Comment:
Since we built on ubuntu, the shared libraries don't work unless we also run on Ubuntu. We kind of have to pick one distro to support first, might as well be ubuntu. That's what Postgres ML is doing.